frontend/src/main/kotlin/org/flaxo/frontend/component/CourseSettingsComponent.kt

```diff
@@ -107,6 +110,17 @@ private class CourseSettingsComponent(props: CourseSettingsProps)
         )
     }
 
+    private fun RBuilder.plagiarismAnalysisFilePatternsInput() {
+        inputComponent(inputId = coursePlagiarismAnalysisFilePatternsSetting,
+                name = "File patterns for plagiarism analysis",
+                description = "Regular expression of the names of files that will be downloaded from the repository " +
```

Comment:
What do you think about the following description. I suppose it's a little bit abstract which is good.
```
Regular expression which will be used to filter repository files for plagiarism analysis. Leave the pattern blank to use a default one (course language source file extensions).